composer.json

```diff
@@ -4,17 +4,17 @@
     "license": "MIT",
     "type": "library",
     "require": {
-        "php": "^5.6|^7.0",
-        "doctrine/dbal": "^2.2",
+        "php": "^7.2",
```

Comment:
doctrine/migrations 2.0 only requires 7.1 so you could lower the restriction here if you wanted to.

Reply:
Hi, I had initially set this to ^7.1 but some of the dependencies required >=7.2. I'm happy to support ^7.1 but I couldn't actually get the dependencies to resolve.

I'll take a look into the config options, 👍for pointing that out.